notesserver/settings/common.py

```diff
@@ -83,3 +83,6 @@
 )
 
 DEFAULT_NOTES_PAGE_SIZE = 25
+
+### Maximum number of allowed notes per course ###
+MAX_ANNOTATIONS_PER_COURSE = 500
```

Comment:
The other settings use the name notes, so lets call this `MAX_NOTES_PER_COURSE `. Also add "for each student" to the comment above.